glue/clients/tests/test_scatter_client.py

```diff
@@ -74,6 +77,9 @@ def check_ticks(self, axis, is_log, is_cat):
         elif is_cat:
```

Comment:
There are a few places in the codebase like this now, where we have 3 code paths (one for datetimes, one for categories, one for others). This is a clue that we should probably be pulling this logic into methods on `Component`, which each component subtype overrides. @aak65 this isn't something that you necessarily need to worry about, but @astrofrog and I should think about it.